pyproject.toml

```diff
@@ -0,0 +1,41 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
```

Comment:
Preferred classifier here @ZLLentz? I'd guess production

Reply:
Development Status :: 5 - Production/Stable is probably the most accurate